queries/xdai/types.ts

```diff
@@ -0,0 +1 @@
+export type BlockDataType = Record<string, { number: string }[]>;
```

Comment:
I think we shouldn't export this from here. It's not going to be used outside `ActivityChart` for now. Types will be different in both solutions.

Reply:
Yeah with the suggested changes you've made I already removed this from here <3